README.md

```diff
@@ -8,6 +8,18 @@ This module adds enterprise grade monitoring for Node.js in PaaS environments th
 * Click on "Set up PaaS Integration"
 * Generate a PaaS token
 * Run `$ npm install --save @dynatrace/oneagent` in your project directory
+* Using the previously created credentials add the following code block as first statement to your application
+
+ ```js	
+try {	
+  require('@dynatrace/oneagent')({	
+    environmentid: '<environmentid>',	
+    apitoken: '<paastoken>',	
```

Comment:
should we also mention here how to configure a custom server? I think this is a mandatory setting for dynatrace managed users.

Reply:
Yes you are right - could you add all possible settings?

Reply:
I added `endpoint`. Please let me know if I should add more or change the wording.